src/library_async.js

```diff
@@ -723,6 +724,9 @@ mergeInto(LibraryManager.library, {
             asyncFinalizers.forEach(function(func) {
               func(asyncWasmReturnValue);
             });
+            Asyncify.sleepCallbacks.forEach(function(func) {
```

Comment:
Why qualify this with `Asyncify.` but not `asyncFinalizers` three lines above?

Reply:
`asyncFinalizers` is captured to a local variable because we clear it out. Those finalizer calls are executed once. The `sleepCallbacks` on the other hand happen constantly, every time.